### ISSUE TITLE
For training on GPU, except for the first trials, other training speeds are very slow

### ISSUE THREAD

**Issue by kaixinbaba** [keras-team/autokeras#1098]
### Bug Description
Except for the first trials, other training speeds are very slow and I feel that I do n’t need a GPU.
The first trials, one step only takes 7 seconds, but the subsequent trials, one step takes more than 50 seconds

### Bug Reproduction
Code is the demo for ImageClassifier
```python
from tensorflow.keras.datasets import mnist
(x_train, y_train), (x_test, y_test) = mnist.load_data()
print(x_train.shape) # (60000, 28, 28)
print(y_train.shape) # (60000,)
print(y_train[:3]) # array([7, 2, 1], dtype=uint8)

import autokeras as ak

# Initialize the image classifier.
clf = ak.ImageClassifier(max_trials=10) # It tries 10 different models.
# Feed the image classifier with training data.
clf.fit(x_train, y_train,epochs=10)
```

Data used by the code:

### Expected Behavior
In my console:
The first trials
```
In [34]: clf.fit(x_train, y_train,epochs=10)
Train for 1500 steps, validate for 375 steps
Epoch 1/10
1500/1500 [==============================] - 8s 5ms/step - loss: 0.1778 - accuracy: 0.9460 - val_loss: 0.0713 - val_accuracy: 0.9793
Epoch 2/10
1500/1500 [==============================] - 7s 5ms/step - loss: 0.0777 - accuracy: 0.9759 - val_loss: 0.0511 - val_accuracy: 0.9851
Epoch 3/10
1500/1500 [==============================] - 7s 5ms/step - loss: 0.0612 - accuracy: 0.9809 - val_loss: 0.0460 - val_accuracy: 0.9870
Epoch 4/10
1500/1500 [==============================] - 7s 5ms/step - loss: 0.0533 - accuracy: 0.9833 - val_loss: 0.0458 - val_accuracy: 0.9873
Epoch 5/10
1500/1500 [==============================] - 7s 5ms/step - loss: 0.0472 - accuracy: 0.9845 - val_loss: 0.0403 - val_accuracy: 0.9888
Epoch 6/10
1500/1500 [==============================] - 7s 5ms/step - loss: 0.0414 - accuracy: 0.9870 - val_loss: 0.0436 - val_accuracy: 0.9883
Epoch 7/10
1500/1500 [==============================] - 7s 5ms/step - loss: 0.0377 - accuracy: 0.9875 - val_loss: 0.0436 - val_accuracy: 0.9883
Epoch 8/10
1500/1500 [==============================] - 7s 5ms/step - loss: 0.0355 - accuracy: 0.9882 - val_loss: 0.0425 - val_accuracy: 0.9891
Epoch 9/10
1500/1500 [==============================] - 7s 5ms/step - loss: 0.0335 - accuracy: 0.9894 - val_loss: 0.0456 - val_accuracy: 0.9884
Epoch 10/10
1500/1500 [==============================] - 7s 5ms/step - loss: 0.0298 - accuracy: 0.9900 - val_loss: 0.0443 - val_accuracy: 0.9883
[Trial complete]
```
but after that
```
Train for 1500 steps, validate for 375 steps
Epoch 1/10
1500/1500 [==============================] - 65s 43ms/step - loss: 0.2302 - accuracy: 0.9355 - val_loss: 0.0940 - val_accuracy: 0.9745
Epoch 2/10
 431/1500 [=======>......................] - ETA: 37s - loss: 0.1870 - accuracy: 0.9558
...
```
### Setup Details
Include the details about the versions of:
 - OS type and version: MacOS 10.15
 - Python: 3.6
 - autokeras: 1.0.2
 - keras-tuner: 1.0.1
 - scikit-learn: 0.22.2.post1
 - numpy: 1.18.1
 - pandas: 1.0.1
 - tensorflow: 2.1.0
 - tensorflow-gpu: 2.1.0

### Additional context
Please help~

**Comment by kaixinbaba** [comment]
Add:
I did not wait for the second trial to finish before the test, so I came to a conclusion.
But now I have tested many times, no matter how much I set **max_trials**
No matter how much **epoch** is set, `ak` will only run **4** trials, and in the **third** trial
Will print this sentence ```INFO: tensorflow: Oracle triggered exit```
Only the **second** of these four trials is slow, and the other three are fast. What is the principle?
Is the second slowness a bug? Under what circumstances will the phrase "Oracle triggered exit" be triggered, leading to the end of the entire training early
